src/index.js

```diff
@@ -174,7 +178,11 @@ export default class extends Component {
     autoplayTimeout: 2.5,
     autoplayDirection: true,
     index: 0,
-    onIndexChanged: () => null
+    onIndexChanged: () => null,
+
+    showAdjacentViews: false,
+    adjacentViewsWidth: 8,
```

Comment:
if default value for showAdjacentViews is false, default for adjacentViewsWidth & adjacentViewsPadding should be 0 right ?

Reply:
They will be ignored, don't worry.

Reply:
Yes, agreed they are ignored, but let's keep some value if they make any sense otherwise let's just keep 0